### PR TITLE
[PVR] Fix crash in CPVRGUIActions::ProcessMenuHooks.

### PR DESCRIPTION
**Pull request by ksooo**
@FernetMenta for review?

### CODE REVIEW

File: xbmc/pvr/PVRGUIActions.cpp [hunk 1]
```diff
@@ -1203,35 +1203,38 @@ namespace PVR
...
-        menuCategory = PVR_MENUHOOK_EPG;
+        if (item->GetEPGInfoTag()->HasPVRChannel())
+        {
+          iClientID = item->GetEPGInfoTag()->ChannelTag()->ClientID();
+          menuCategory = PVR_MENUHOOK_EPG;
+        }
+        else
+          return false;
+      }
+      else if (item->IsPVRChannel())
```
Comment (FernetMenta) [inline xbmc/pvr/PVRGUIActions.cpp]: now it will segfault here :)

Comment (FernetMenta) [inline xbmc/pvr/PVRGUIActions.cpp]: @ksooo sorry, I misinterpreted the diff

**Review by FernetMenta — APPROVED** [review]
thanks

**Comment by ksooo** [comment]
Jenkins build this please

**Comment by ksooo** [comment]
jenkins build this please

**Pull request merged**